fixtures/cameo/root-par-6.json

```diff
@@ -0,0 +1,753 @@
+        },
+        {
+          "dmxRange": [202, 207],
+          "type": "ColorPreset",
+          "comment": "User Color_1"
+        },
+        {
+          "dmxRange": [208, 213],
+          "type": "ColorPreset",
+          "comment": "User Color_2"
+        },
+        {
+          "dmxRange": [214, 219],
+          "type": "ColorPreset",
+          "comment": "User Color_3"
+        },
+        {
+          "dmxRange": [220, 225],
+          "type": "ColorPreset",
+          "comment": "User Color_4"
+        },
+        {
+          "dmxRange": [226, 255],
+          "type": "ColorPreset",
+          "comment": "UV"
+        }
+      ]
+    },
+    "Amber": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "UV": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "UV"
+      }
+    },
+    "Strobe 2": {
+      "name": "Strobe",
+      "capabilities": [
+        {
+          "dmxRange": [0, 5],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [6, 255],
+          "type": "StrobeSpeed",
+          "speedStart": "1Hz",
+          "speedEnd": "20Hz"
+        }
+      ]
+    },
+    "Dimmer 4": {
+      "name": "Dimmer",
+      "capability": {
+        "type": "Intensity",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Dimmer fein": {
+      "capability": {
+        "type": "Intensity"
+      }
+    }
+  },
+  "modes": [
+    {
+      "name": "3-Channel",
+      "shortName": "3ch",
+      "channels": [
+        "Dimmer",
+        "Color Presets",
+        "DMX Delay"
+      ]
+    },
+    {
+      "name": "5-Channel #1",
+      "shortName": "5ch#1",
+      "channels": [
+        "Dimmer 2",
+        "Strobe Multifuntional",
+        "Color Presets",
+        "Sound Sensitivity",
+        "DMX Delay"
+      ]
+    },
+    {
+      "name": "5-Channel #2 (compatibility for ROOT PAR 4)",
+      "shortName": "5ch#2",
+      "channels": [
+        "Red",
+        "Green",
+        "Blue",
+        "White",
+        "DMX Delay"
+      ]
+    },
+    {
+      "name": "6-Channel",
+      "shortName": "6ch",
+      "channels": [
+        "Dimmer 3",
+        "Strobe",
+        "Chase",
+        "Color Presets 1",
+        "Color Presets 2",
+        "DMX Delay"
+      ]
+    },
+    {
+      "name": "7-Channel",
+      "shortName": "7ch",
+      "channels": [
+        "Red",
+        "Green",
+        "Blue",
+        "White",
+        "Amber",
+        "UV",
+        "DMX Delay"
+      ]
+    },
+    {
+      "name": "9-Channel",
+      "shortName": "9ch",
+      "channels": [
+        "Dimmer",
+        "Strobe 2",
+        "Red",
+        "Green",
+        "Blue",
+        "White",
+        "Amber",
+        "UV",
+        "DMX Delay"
+      ]
+    },
+    {
+      "name": "12-Channel",
+      "shortName": "12ch",
+      "channels": [
+        "Dimmer 4",
+        "Dimmer fein",
```

Comment:
Typo but dependant on possible changes in Definition in Previous Review comment.